repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -385,6 +385,11 @@ export interface GroupPassengerType {
     companions: CompanionInfo[];
 }
 
+export interface ActualPassengerType {
```

Comment:
Minor: not sure about the name - IndividualPassengerType maybe?

Reply:
See @matt-twomey's suggestion above.